ldi-core/ldi-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/rdf/formatter/LdiRdfWriter.java

```diff
@@ -2,11 +2,34 @@
 
 import org.apache.jena.rdf.model.Model;
 import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFWriterBuilder;
 
+import java.io.OutputStream;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.rdf.formatter.PrefixAdder.addPrefixesToModel;
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
 public interface LdiRdfWriter {
-	String write(Model model);
+
+	RDFWriterBuilder builder();
+
+	/**
+	 * @param model Model to be parsed to a String
+	 * @return String representation of the RDF Jena model in the RDF format provided in the LdiRdfWriterProperties
+	 */
+	default String write(Model model) {
```

Comment:
Default methods in interfaces are not only bad practice when you are not dealing with legacy code, as you are breaking the concept of the interfaces. In this case you are also breaking the strategy pattern. Can you think of more elegant solution?